open-craft/openedx-completion-aggregator
# Rewire completion v0 api to use completion-aggregator infrastructure

**Description:** Rewire completion v0 api to use completion-aggregator infrastructure

**JIRA:** OC-4522

**Dependencies:**  OC-4528

**Merge deadline:** None

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

Check output of: /api/completion-aggregator/v0/course/
And compare to: /api/completion/v0/course/

Check output of: /api/completion/v0/course/course-course-v1:edX+DemoX+Demo_Course/
And compare to: /api/completion-aggregator/v0/course/course-v1:edX+DemoX+Demo_Course/

**Reviewers:**

- @jcdyer 

**Merge checklist:**

**Post merge:**
      finished.

**Author concerns:** Missing migration of [CompletionBlockUpdateView](https://github.com/edx-solutions/edx-platform/blob/master/lms/djangoapps/completion_api/views.py#L441). @jcdyer shouldn't this be on the completion api? instead of here?
